packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -58,7 +58,8 @@ void main() {
     expect(find.text('Delete'), findsNothing);
   });
 
-  testWidgets('Dialog not barrier dismissible by default', (WidgetTester tester) async {
+  testWidgets('Dialog not barrier dismissible by default',
+      (WidgetTester tester) async {
```

Comment:
A lot of the changes here seem to be style changes. Did you manually adjust these or are you using a linter? Would you remind reverting most of them? I appreciate wanting to clean up the code, but ideally we'd want as few things unconnected to the issue as possible in this PR. Most of the changes splitting a line into multiple do not feel necessary and in a couple places arguably look worse.